python/cudf/dataframe/series.py

```diff
@@ -449,12 +462,12 @@ def _binaryop(self, other, fn, reflect=False):
             return other._apply_op(self, fn)
 
         nvtx_range_push("CUDF_BINARY_OP", "orange")
-
+        result_name = utils.get_result_name(self, other)
         lhs, rhs = _align_indices(self, other)
         rhs = self._normalize_binop_value(rhs)
         outcol = lhs._column.binary_operator(fn, rhs, reflect=reflect)
         result = lhs._copy_construct(data=outcol)
-        result.name = None
+        result._name = result_name
```

Comment:
Any reason to not just pass the name into the `_copy_construct` call above?

Reply:
Straight away made that change of variable assignment but You are right, I can pass it to `_copy_construct`, made this change.